workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/static_report.py

```diff
@@ -439,7 +430,37 @@ def main(args):
     }
 
     for filename, html in pages.items():
-        upload(html, os.path.join(args.output, filename))
+        upload(html, os.path.join(output, filename))
+
+
+def register_parser(subparsers):
+    parser = subparsers.add_parser("report", help="Generate a static html report.")
+    parser.add_argument("input", help="Directory containing multiple run diagnostics.")
+    parser.add_argument("output", help="Location to save report html files.")
+    parser.set_defaults(func=main)
+
+    parser = subparsers.add_parser(
+        "report-from-urls",
+        help="Generate a static html report from list of diagnostics.",
+    )
+    parser.add_argument(
+        "inputs",
+        help="Folders containing diags.nc. Will be labeled with "
+        "increasing numbers in report.",
+        nargs="+",
+    )
+    parser.add_argument("-o", "--output", help="Location to save report html files.")
```

Comment:
```suggestion
    parser.add_argument("output", help="Location to save report html files.")
```
since it is a required argument, I suggest making it positional. If you feel strongly about keeping the `-o`, add `required=True` as a kwarg.

Reply:
Good point. I will go for `required=True` to disambiguate from the variadic position arguments.